source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,49 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-04-21
+
+### Highlights
+
+#### Apps Framework (Developer Preview)
+ - This feature is available on Mobile Apps when Apps Framework becomes available on Mattermost v5.35.0 and the proxy plugin is loaded on an instance. The launch for the Developer Preview of the Apps Framework is scheduled for April 29th.
```

Comment:
```suggestion
 - Apps Framework is a new way to integrate with external tools, and allows developers to create interactive Apps in Mattermost, using any development language they're comfortable with. The new Apps work seamlessly across mobile and desktop clients.  This is a developer preview and is not intended for production instances of Mattermost yet.  This feature will be available for self-managed customers when Mattermost v5.35.0 ships and the Apps Framework Plugin is loaded on an instance, before then developers can use the `cloud` branch to get a local test environment running. The launch for the Developer Preview of the Apps Framework is scheduled for April 29th.  Learn more: 
```

Reply:
@justinegeffen - do we have the URL for the new Apps Framework docs location?  (want to insert it here)

Reply:
It should be located at https://developers.mattermost.com/apps/getting-started/ once it's live but if we could hold off on that until I've ironed out some issues that'd be great.